.storybook/Hug.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Hug, Card, Text } from '../src'
+
+const text = (
+  <Text.span>
+    Today's best deal! You save <Text.span bold>$300</Text.span> for your party
```

Comment:
Pro tip: If you use an apostrophe in JSX, GitHub will not do this nonsense `’`

Reply:
Thanks for the life hack